### PR TITLE
Add description of architecture and diagram

### PR DESCRIPTION
**Pull request by Kechr**

### CODE REVIEW

File: project_report/tex/architecture.tex [hunk 1]
```diff
@@ -1 +1,38 @@
 \section{Architecture}
+When the project case was presented to the group it was discussed what
+subsystems the MES should consist of. It was clear from the start that an OPC-UA
+client was needed, as this should connect with the beer production machine. This
```
Comment (Bokaiii) [inline project_report/tex/architecture.tex]: with = to

File: project_report/tex/architecture.tex [hunk 2]
```diff
@@ -1 +1,38 @@
...
+client should in some way store the collected data from the production machine,
+which means that some kind of database should be a part of the MES. As the data
+would only consist of simple data types, integers, strings, etc., it was decided
+to use a relational database. This way, data from different batches could easily
+be stored in a way so the relations between the data could be kept as needed. By
```
Comment (Bokaiii) [inline project_report/tex/architecture.tex]: ... be stored in such a way that the relations....

File: project_report/tex/architecture.tex [hunk 3]
```diff
@@ -1 +1,38 @@
...
+having a database, the subsystems does not have to store data locally, thus
```
Comment (Bokaiii) [inline project_report/tex/architecture.tex]: Indtil nu har resten af sektionen været i datid (does)

does = would

File: project_report/tex/architecture.tex [hunk 4]
```diff
@@ -1 +1,38 @@
...
+making the same data available for other parts of the system, securing data
+consistency. \\
+
+It was discussed whether to develop a desktop client functioning as a dashboard
+to control the production machine, or if it should be a web client. The decision
+landed on a web client. A web client is a more simple solution for the dashboard
+as it can run in all supported web browsers. \myworries{write a more valid
+reason here please :)}. \\
```
Comment (Bokaiii) [inline project_report/tex/architecture.tex]: \myworries{Don't forget about more valid reasons}

File: project_report/tex/architecture.tex [hunk 5]
```diff
@@ -1 +1,38 @@
...
+
+To make the data from the database available for all subsystems in the MES,
```
Comment (Bokaiii) [inline project_report/tex/architecture.tex]: available for = available to

File: project_report/tex/architecture.tex [hunk 6]
```diff
@@ -1 +1,38 @@
...
+consistency. \\
+
+It was discussed whether to develop a desktop client functioning as a dashboard
+to control the production machine, or if it should be a web client. The decision
+landed on a web client. A web client is a more simple solution for the dashboard
+as it can run in all supported web browsers. \myworries{write a more valid
+reason here please :)}. \\
+
+To make the data from the database available for all subsystems in the MES, 
+it was decided to develop a REST API. The REST API that acts as a translator
```
Comment (Bokaiii) [inline project_report/tex/architecture.tex]: Remove that (The REST API that)
Again switched to nutid

File: project_report/tex/architecture.tex [hunk 7]
```diff
@@ -1 +1,38 @@
...
+between the subsystems in the MES, therefore simplifying the data management.
+And example could be that the dash board needs the data for a specific batch to
```
Comment (Bokaiii) [inline project_report/tex/architecture.tex]: And = an example
Dash board = dashboard

**nutid**

File: project_report/tex/architecture.tex [hunk 8]
```diff
@@ -1 +1,38 @@
...
+generate a batch report. Instead of having the dashboard communicate directly
+with the database, a layer is added, the REST API, which both adds security and
+handles the queries in a uniform way. The database then send a query result to
```
Comment (Bokaiii) [inline project_report/tex/architecture.tex]: send = sends (or rewrite to datid)

**Review by Bokaiii — CHANGES_REQUESTED** [review]
Looks good otherwise

**Review by Joergenson — APPROVED** [review]
added the changes myself ;)

**Pull request merged**